### ISSUE TITLE
[1.6.0-r6 (66)/Sony Z2/6.0.1/Unexpected behaviour occurs when user sign up

### ISSUE THREAD

**Issue by ayaadel87** [Lomotif/lomotif-android-issues#616]
Frequency:Every time
Steps:
Lauch the app
click on skip for now
click on Account icon at the bottom of the app
Enter any email address, then click on next via mobile keyboard
enter any password, then click on Next via mobile Keyboard

Expected Result:
Mobile keyboard should disappear

Actual Result:
Page is hanged and crashed

Attached video for capturing the steps

https://drive.google.com/open?id=0B12VjRgvbAD0SXMyX0xwODhYTC1OOWlRV1pNYW9pRWZDMTZZ

**Comment by testershub** [comment]
@xiuie I can't reproduce this issue for build [1.6.0-r4 (64)] on Samsung Galaxy S6 / 6.0.1 and Samsung Galaxy S4 / 5.0.1.

Only one thing, keypad is not disappearing when user taps on "Next" option on keypad from "Password" field.

**Comment by ayaadel87** [comment]
@testershub : can you check the video i have uploaded

**Comment by testershub** [comment]
@ayaadel87 Yes, I have referred the attached video.
Application is not crashing. Only keypad is not disappearing.

**Comment by ayaadel87** [comment]
@testershub : yes keypad is not disappearing and user can go nowhere, so page is hanged
